index.html

```diff
@@ -80,7 +80,7 @@ <h2>
                                 the right ingredients <br/>
                                 for the right food
                             </h2>
-                            <img src="/images/divider-white.png" alt="divider-white">
+                            <img src="images/divider-white.jpg" alt="divider-white">
```

Comment:
Here are my comments to index.html:
- I can see that you started to use BEM naming convention which is cool
- line 25: do you need this div?
- line 30: do you need this div?
- line 78: do you need so many nested divs? Perhaps you can create one/two and add multiple classes to it, like `row col-12 caption` for example
- line 84: nice class naming!
- line 134: perhaps an article?
- line 207: don't need to add spaces at the beginning/end, it doesn't make any difference and may be confusing. Here and in other places
- line 329: I guess this is a part of the article
- line 328,351: same as mentioned above - do you need so many nested divs?
- line 370: I guess this title should be inside the article. Also do you need two articles? Perhaps one article with two paragraphs `<p>`
- line 442: As mentioned in previous review - I personally would rather use a class instead of `strong`
- line 425: I would rather placed a `footer` outside of `main`, as you did with `header`